students/yujeong/users/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+class User(models.Model) :
+    user_name         = models.CharField(max_length = 20)
+    user_email        = models.EmailField(max_length = 45)
+    user_id           = models.CharField(max_length = 50)
+    user_pw           = models.CharField(max_length = 20)
+    user_phone_number = models.CharField(max_length = 50)
```

Comment:
1. 속성명 앞에 user_는 모두 제거 해주세요. User table에 있는 속성이기 때문에 없어도 충분히 의미 전달이 됩니다.
2. pw 처럼 변수명을 줄여서 사용하는 것은 지양해 주세요.
3. email 필드에는 unique 속성을 추가하여 중복 저장을 방지해 주세요.
4. id 가 우리가 일반적으로 생각하는 그 ID 라면 username을 ID의 역할로 많이 사용 됩니다.
5. password의 max_length는 추후 암호화한 값이 들어갈 예정이기 때문에 200으로 늘려 주세요.
```suggestion
    name         = models.CharField(max_length = 20)
    email        = models.EmailField(max_length = 45)
    username     = models.CharField(max_length = 50)
    password     = models.CharField(max_length = 20)
    phone_number = models.CharField(max_length = 50)
```